Packs/ExabeamDataLake/Integrations/ExabeamDataLake/README.md

```diff
@@ -0,0 +1,159 @@
+Cloud-delivered Data Lake provides a highly scalable, cost-effective, and searchable log management system. Data Lake is used for log collection, storage, processing, and presentation.
+This integration was integrated and tested with version LMS-i40.3 of Exabeam Data Lake.
+
+## Configure Exabeam Data Lake on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Exabeam Data Lake.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Server URL |  | True |
+    | User Name |  | True |
+    | Password |  | True |
+    | Cluster Name | The default value is usually 'local', suitable for standard setups. For custom cluster deployments, consult Exabeam Support Team. | True |
+    | Trust any certificate (not secure) |  |  |
+    | Use system proxy settings |  |  |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## Commands
+
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+
+### exabeam-data-lake-search
+
+***
+Get events from Exabeam Data Lake.
+
+#### Base Command
+
+`exabeam-data-lake-search`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| query | The search query string to filter the events by. (e.g. "VPN". See more examples in the integration description). | Required | 
+| start_time | The starting date for the search range. The search range should be at least one day long and can extend up to a maximum of 10 days. | Required | 
+| end_time | The ending date for the search range. This defines the end of the search range, which should be within one to ten days after the start_time. | Required | 
+| limit | The maximal number of results to return. Maximum value is 3000. | Optional | 
+| page | The page number for pagination. | Optional | 
+| page_size | The maximal number of results to return. Maximum value is 3000. | Optional | 
```

Comment:
Is this description correct?
Should it be: The maximum number of results to return per page.
Is the maximum value correct?